docs/src/stories/OneHundredKRows.js

```diff
@@ -61,7 +61,10 @@ class Story extends React.PureComponent {
                     data={data}
                     columns={columns}
                     defaultPageSize={3}
-                    showPagination={false}
+                    showPagination={{
```

Comment:
Please remove the custom pagination settings from all stories except for the pagination story meant to demo it :)